doc/conf.py

```diff
@@ -32,6 +32,7 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
```

Comment:
Yes napoleon is inside sphinx. numpydoc is an extra dependency. They do the same thing